tests/utimensat/00.t

```diff
@@ -0,0 +1,32 @@
+#! /bin/sh
+# $FreeBSD$
+
+desc="utimensat changes timestamps on any type of file"
+
+dir=`dirname $0`
+. ${dir}/../misc.sh
+echo "1..32"
+
+n0=`namegen`
+n1=`namegen`
+
+expect 0 mkdir ${n1} 0755
+cdir=`pwd`
+cd ${n1}
```

Comment:
This should probably be `expect`-ed so we're sure that we're going to the proper directory before and after the test and not just littering temporary files where the test is being run from... whether or not pjdfstest does this 100% of the time elsewhere though is a good question. If not, then it should be fixed everywhere in a separate commit.